src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx

```diff
@@ -96,33 +84,37 @@ const ClusterDashboard = () => {
       longStringShorten(getClusterHash(cluster.operators, accountAddress).slice(2), 4),
       <Grid container style={{ gap: 8 }}>
         {cluster.operators.map((operator: any, index: number) => {
-          return <Grid item
-                       container
-                       key={index}
-                       onMouseLeave={handleGridLeave}
-                       className={classes.CircleImageOperatorWrapper}
-                       onMouseEnter={() => handleGridHover(getClusterHash(cluster.operators, accountAddress) + operator.id)}
-          >
-            {(hoveredGrid === getClusterHash(cluster.operators, accountAddress) + operator.id) && (
-              <OperatorCard operator={operator}/>
-            )}
-            <OperatorCircleImage operatorLogo={operator.logo}/>
-          </Grid>;
+          return (
+            <Grid
+              item
+              container
+              key={index}
+              onMouseLeave={handleGridLeave}
+              className={classes.CircleImageOperatorWrapper}
+              onMouseEnter={() => handleGridHover(getClusterHash(cluster.operators, accountAddress) + operator.id)}
+            >
+              {hoveredGrid === getClusterHash(cluster.operators, accountAddress) + operator.id && <OperatorCard operator={operator} />}
+              <OperatorCircleImage operatorLogo={operator.logo} />
+            </Grid>
+          );
         })}
       </Grid>,
       cluster.validatorCount,
-      isNaN(cluster.runWay) ? (
-        <NaDisplay text={translations.NA_DISPLAY.TOOLTIP_TEXT} tooltipClassExtend={classes.TooltipCustomSize}/>
-      ) : remainingDays,
-      <ClusterWarnings cluster={cluster}/>,
+      isNaN(cluster.runWay) ? <NaDisplay text={translations.NA_DISPLAY.TOOLTIP_TEXT} tooltipClassExtend={classes.TooltipCustomSize} /> : remainingDays,
+      <ClusterWarnings cluster={cluster} />
     );
   });
 
   const openSingleCluster = (listIndex: number) => {
-    processStore.setProcess({
-      processName: 'single_cluster',
-      item: sortedClusters[listIndex],
-    }, 2);
+    dispatch(
```

Comment:
this looks redundant cause in next line there is setting selected cluster in redux